src/modules/NeighborInfoModule.cpp

```diff
@@ -138,13 +138,19 @@ Pass it to an upper client; do not persist this data on the mesh
 */
 bool NeighborInfoModule::handleReceivedProtobuf(const meshtastic_MeshPacket &mp, meshtastic_NeighborInfo *np)
 {
+    LOG_DEBUG("NeighborInfo: handleRecievedProtobuf");
     if (np) {
         printNeighborInfo("RECEIVED", np);
         updateNeighbors(mp, np);
     } else if (mp.hop_start != 0 && mp.hop_start == mp.hop_limit) {
         // If the hopLimit is the same as hopStart, then it is a neighbor
         getOrCreateNeighbor(mp.from, mp.from, 0, mp.rx_snr); // Set the broadcast interval to 0, as we don't know it
     }
+    if (mp.decoded.want_response && np->neighbors_count == 0) { // want response, and is otherwise empty.
```

Comment:
`want_response` packets are handled in `callModules()`. The cleanest way to implement this is by overriding `allocReply()`. See e.g. the NodeInfoModule for an example. In this way, the correct `request_id` is also set, which the recipient uses to link to the request.

Reply:
Done. 
Needs clean up and some further testing, but it seems to work.

And I'll need to change the way I trigger it to avoid the empty array